agrona/src/main/java/org/agrona/CloseHelper.java

```diff
@@ -56,4 +60,54 @@ public static void close(final AutoCloseable closeable)
             LangUtil.rethrowUnchecked(e);
         }
     }
+
+    /**
+     * Close all closeables in closeables. If any of them throw then throw that exception.
+     * If multiple closeables throw an exception when being closed, then throw an exception that contains
+     * all of them as suppressed exceptions.
+     *
+     * @param closeables to be closed.
+     */
+    public static void closeAll(final List<? extends AutoCloseable> closeables)
+    {
+        if (closeables == null)
+        {
+            return;
+        }
+
+        final List<Exception> exceptions = new ArrayList<>();
```

Comment:
It would be better if this was not allocated unless needed.

Reply:
Fixed.